### DISCUSSION TITLE
torch dependency is over 700MB

### DISCUSSION THREAD

**Discussion by guest271314** [coqui-ai/TTS#1412] [General]
Is there a way to build TTS without the 700MB torch dependency? If you happen to be on a machine with minimal disk space that dependency is restrictive.

**Comment by erogol** [comment]
Nope, it is not possible. Torch is a core dependency for 🐸TTS

**Reply by guest271314** [reply]
Which specific files are necessary from torch? Is there any way to decrease the size of the files needed to compile torch with precisely only the file TTS requires?
